### DISCUSSION TITLE
slow download

### DISCUSSION THREAD

**Discussion by BoogeyMaan** [shaked6540/YoutubePlaylistDownloader#57] [Q&A]
where is the ip address implement. every time it connect 45.113.133.76 and download speed very slow 300kbps.

**Comment by shaked6540** [comment]
I'm not sure what you mean, the program does not connect to a specific ip address, its just downloading whatever youtube is giving it. 
If the downloads are slow its either because your internet is slow, or youtube is limiting the connection speed
